test-e2e/k8spod.py

```diff
@@ -14,3 +15,76 @@ def ping(self, ip):
             return False
         return True
 
+    def http(self, client_name):
```

Comment:
Please use the same functions name, in https://github.com/futurewei-cloud/mizar/blob/dev-next/test/trn_controller/endpoint.py
Here do_httpd.